### ISSUE TITLE
No Documentation

### ISSUE THREAD

**Issue by yaskor** [metalmatze/drone-cron#5]
Would love to use this but there is no documentation.

How do I achieve nightly builds?
Does this work with 0.8?

**Comment by metalmatze** [comment]
Yes. It works with 0.8. There's no real documentation as this is just a simple tool that I wrote in an hour.
Please check the example on how to run something every midnight: https://github.com/metalmatze/drone-cron/blob/master/examples/config.yaml

AFAIK cron jobs are probably coming to drone in 0.9.
